### ISSUE TITLE
Warn user if he doesn't have English filter enabled

### ISSUE THREAD

**Issue by vbagirov** [AdguardTeam/ReportsWebApp#82]
On step 4, if the user has selected any filters, but not English filter, show a warning in red:
https://uploads.adguard.com/vbagirovceqqf.png

In other words:

 * No filter selected --> No warning
 * English filter + any other filters selected --> No warning
 * Some filters selected, but no English filter --> Show warning

**Comment by TPS** [comment]
Per https://github.com/AdguardTeam/AdguardFilters/issues/3937#issuecomment-308451213, I propose you rename the filterlist "English" → "Base" while implementing this.

**Comment by vbagirov** [comment]
@TPS we'd like to wait a little bit more on this change.

**Comment by TPS** [comment]
Maybe update @ least the description, for now?  I could do the PR for _that_, if you wish, but this gets more important, especially since RFEs like https://github.com/AdguardTeam/AdguardForAndroid/issues/1579.

**Comment by vbagirov** [comment]
@TPS I don't think this issue is a proper place to discuss the renaming of English filter. And when we change its name, we will change it everywhere more or less at once. Changing it in the web reporting tool won't take much time anyway, as we don't have to update it on the client's side.

**Comment by vbagirov** [comment]
Referencing #150 

>When the user is entering a website, the report should check the site's language and if it is e. g. Japanese and the user is not selecting Japanese filter, he should get a notice to try to activate Japanese filter and check again.

**Comment by TPS** [comment]
Now that the name is changed to "AdGuard Base filter," could this issue title be updated?  TIA! 🙇🏾‍♂️
